### PR TITLE
remove old plugin and added plugin manager

### PR DESCRIPTION
**Pull request by nxft**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jmeter.rb [hunk 1]
```diff
@@ -11,34 +11,9 @@ class Jmeter < Formula
...
-  resource "jmeterplugins-webdriver" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-WebDriver-1.4.0.zip"
-    sha256 "521c2f7d452a84099407534bd50f29fd3761aa8a5beca52966bb9731e33b03e2"
-  end
-
-  resource "jmeterplugins-hadoop" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-Hadoop-1.4.0.zip"
-    sha256 "93030738d613748a685764fbfff0fe00ad2e161f2b72df6365294adc88db09b4"
+  resource "jmeter-plugins-manager" do
+    url "https://jmeter-plugins.org/get/"
```
Comment (bayandin) [inline Formula/jmeter.rb]: Use `https://search.maven.org/remotecontent?filepath=kg/apc/jmeter-plugins-manager/1.3/jmeter-plugins-manager-1.3.jar` plase

**Review by bayandin — CHANGES_REQUESTED** [review]
It looks like these changes will "break" `jmeter` after update (users expect a certain amount of plugins installed, could use some them in their scripts). Not sure what is the best way to handle it.

- also, revision bump is required
- commit message should be something like `jmeter: replace plugins with plugin manager`

File: Formula/jmeter.rb (hunk 1 above)
Comment (NonoXie) [reply Formula/jmeter.rb]: sure!
how do you find that url?

Comment (bayandin) [reply Formula/jmeter.rb]: https://jmeter-plugins.org/get/ gives you a file from https://maven.org, so I just searched for `jmeter-plugins-manager` on https://search.maven.org/

**Comment by NonoXie** [comment]
> It looks like these changes will "break" `jmeter` after update (users expect a certain amount of plugins installed, could use some them in their scripts). Not sure what is the best way to handle it.

but the plugins are old and influences some feature with newer JDK, and also the plugins manager should be the proper way to manage these plugins.

**Pull request closed without merging**